CandidateReranking.md

```diff
@@ -0,0 +1,369 @@
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| name | String | **FALSE** | Name of the keyword, e.g. "Java".
+| type | String | **FALSE** | Type of the keyword, e.g. "skill".
+| weight | Integer | **FALSE** | Weight of the keyword as an int. \[0-100\]
+| relationships | [Relationship](#relationship) | **FALSE** | Array of relationship objects which enrich the keyword with additional entities.
+
+#### Relationship
+A `relationship` is composed of a **\<semantic enrichment type\>** and the associated set of entities which can be classified under that **\<semantic enrichment type\>**. Accepted values for the **\<semantic enrichment type\>** are: `job_titles`, `occupations`, `skills`, `job_level`, `text_kernel_related_keywords`, `interesting_keywords`, `related_keywords`, `related_keywords_recruiter`, `related_keywords_jobseeker`, `raw_job_titles_jobseeker`, `raw_job_titles_recruiter`, `extracted_kewords`, `custom_keywords`, `related_search_terms`, and `job_titles_recruiter`.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| **\<semantic enrichment type\>** | [Entity[]](#entity) | **FALSE** | Relationship to the keyword `name` value.
+
+#### Entity
+An `entity` is composed of a `name`, `weight`, and `selected` boolean. An `entity` enriches the original keyword with terms or phrases which share a close association. For example, if you were to have a `keyword` java, an enrichment entity might be java or programming.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| name | String | **FALSE** | Name of enrichment `entity`. e.g., javascript, java, programming or software engineer.
+| weight | double | **FALSE** | Weight of enriched term according to the Semantic Search API. \[0.0 - 1.0\]
+| selected | boolean | **FALSE** | Specific for the `related_search_terms` Semantic Enrichment. If selected the value is sent to the Reranker as a feature to be extracted for reranking else it is left out of reranking. All entites default to true.
+
+#### Filter
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| id | String | **FALSE** | Unique identfier for the **Filter**.
+| name | String | **FALSE** | Name of the **Filter**.
+| type | String | **FALSE** | Type of **Filter** applied. Accepted values are `facet` or `range`. Service defaults to `facet`.
+| values | String[] | **FALSE** | Values chosen to filter on from the original search request **Filter**.
+
+### Profile
+----------
+The profile is built with data from either **MY_SUPPLY** or **EDGE** (identified in the `source` param). Using data from each profile, the service builds a RerankRequest Document, which is sent along with features extracted from the query to the **Reranker**. The **Reranker** then returns the list of profiles in a new ordering, which should be of more relevance to the query.
```

Comment:
One minor comment on "Using data from each profile". Should it be "Using data from either profile source"? Maybe I am misunderstanding, but there is only one profile source right? You can't have both a my supply and edge profile in one request?

Reply:
It would be good for me to clarify that all profiles are from the same source here or maybe not say anything about the source if that's adding confusion.

Reply:
The list of profiles comes from a single source not both.  And I got into more specific details on the service itself which I don't need to do. I think what's important to understand is that data/features are extracted from the profile to help build the new more relevant results.